### DISCUSSION TITLE
Search in settings works on per-character basis and not per-word, which is inconvenient

### DISCUSSION THREAD

**Discussion by EVAST9919** [ppy/osu#15267] [Q&A] [answered]
Opening a discussion first since I'm not sure if that's intentional or not.
Using search in settings is a bit weird now because it can bring a lot of unrelated results if search query is tiny.
(Also here's an interesting example)
![osu_2021-10-24_14-39-03](https://user-images.githubusercontent.com/22874522/138592678-9e7d28c6-79af-4cfb-935e-0fe36b1a7f85.png)

**Comment by bdach** [comment, marked as answer]
This is a result of https://github.com/ppy/osu-framework/pull/4778.

Maybe that code could be modified to break up the matched text on finding whitespace or something. Or just toggleable entirely and turned off in the case of settings.

**Reply by peppy** [reply]
I think it's probably fine to completely disable for settings. I can't immediately think of a case it would be helpful.

**Reply by peppy** [reply]
Will be addressed via https://github.com/ppy/osu-framework/pull/4842.
